cmd/mimir/help.txt.tmpl

```diff
@@ -432,13 +456,21 @@ Usage of ./cmd/mimir/mimir:
   -query-frontend.query-sharding-total-shards int
     	The amount of shards to use when doing parallelisation via query sharding by tenant. 0 to disable query sharding for tenant. Query sharding implementation will adjust the number of query shards based on compactor shards. This allows querier to not search the blocks which cannot possibly have the series for given query shard. (default 16)
   -query-frontend.results-cache.backend string
-    	Backend for query-frontend results cache, if not empty. Supported values: [memcached].
+    	Backend for query-frontend results cache, if not empty. Supported values: [memcached redis].
```

Comment:
[nit] The other descriptions have it formatted like this:
```suggestion
    	Backend for query-frontend results cache, if not empty. Supported values: memcached, redis.
```

Reply:
Fixed!